NorthstarDLL/mods/modmanager.cpp

```diff
@@ -16,6 +16,7 @@
 #include <string>
 #include <sstream>
 #include <vector>
+#include <regex>
```

Comment:
I'm a bit confused why this include is needed bow but wasn't before ^^"

Reply:
TL;DR: doesn't hurt and can help with preventing issues in case we shuffle the headers around later.